src/ControllerBase.h

```diff
@@ -49,6 +48,17 @@
 #define DEFAULT_CAL_ITERATIONS 3
 #define WATCHDOG_TIMEOUT 30000
 
+#define SENSOR 31855 //MAX31855
```

Comment:
It is much more preferable to have explicit defines for each sensor instead of specifying the chip version, e.g.:
```
#define TEMPERATURE_SENSOR_MAX31855
//#define TEMPERATURE_SENSOR_MAX6675
```

Also, I think it is much better to specify these defines in the `platformio.ini` file so that one can specify different targets there.

Reply:
done. please recheck.